hack/aks/Makefile

```diff
@@ -307,6 +308,7 @@ vnetscale-swift-byocni-nokubeproxy-up: rg-up vnetscale-swift-net-up ## Bring up
 		--no-ssh-key \
 		--os-sku $(OS_SKU) \
 		--kube-proxy-config $(KUBE_PROXY_JSON_PATH) \
+		--pod-ip-allocation-mode StaticBlock \
```

Comment:
Does this work? Have you attempted to create a Static Block cluster with --network-plugin being none. IIRC this would throw an AKS-RP exception previously.

Reply:
yes, pipeline run is also successful: https://msazure.visualstudio.com/One/_build/results?buildId=120192520&view=results